### ISSUE TITLE
Cannot versioning .js files

### ISSUE THREAD

**Issue by rocc-o** [kentaroi/eleventy-plugin-rev#1]
Hello, 
it seems that this plugin installed on my 11ty site is versioning the .css files only. No versioning is applied to .js files. 
Also, it versions all .css files on the page, and not just the modified one. 
And when refreshing the page, the versioning is removed from all .css files. 
Is this intentional or could it be due to some problem at my end?

**Comment by kentaroi** [comment]
The plugin is just a bunch of helper functions to be used from other plugins to make files revisioned.
Admittedly, it is supposed to be `addPlugin()`ed by users in `.eleventy.js`, but it is just because Eleventy does not support plugins' dependencies.

If users use it directly from `.eleventy.js`, it'll hardly work, unless they read some part of Eleventy's source code and understand Eleventy's behaviour.

(Actually, when I started to write the plugin, I intended the plugin to be used by both plugin authors and users. However, now I think it is quite difficult for users with current Eleventy's APIs.)
